### PR TITLE
System

### PR DESCRIPTION
**Pull request by toji59**

### CODE REVIEW

**Review by starascendin — APPROVED** [review]

**Comment by starascendin** [comment]
LGTM

File: contracts/DAOFundingFactory.sol [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./DAOFunding.sol";
+import "./CloneFactory.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract DAOFundingFactory is CloneFactory, Ownable {
+
+    DAOFunding[] public children;
```
Comment (starascendin) [inline contracts/DAOFundingFactory.sol]: Change children to `investments`

File: hardhat.config.ts [hunk 2]
```diff
@@ -69,7 +60,7 @@ export default {
       url: 'https://api.avax-test.network/ext/bc/C/rpc',
       gasPrice: 225000000000,
       chainId: 43113,
-      accounts: [`0x${process.env.FUJI_PRIVATE_KEY}`]
+      accounts: []
```
Comment (barroqt) [inline hardhat.config.ts]: Do not delete the account permission

File: contracts/DAOFunding.sol [hunk 3]
```diff
@@ -0,0 +1,18 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+// TODO(system): implement logic around DAO (deposit, withdraw, distribute)
+contract DAOFunding is Initializable {
```
Comment (starascendin) [inline contracts/DAOFunding.sol]: Suggestion, lets rename using the definitions provided by the prompt:
* DAOFunding -> Investment
* children -> investments

**Review by starascendin — APPROVED** [review]

**Pull request merged**